src/controllers/dynakube/version/image_hash_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/src/controllers/dynakube/version/testdata"
 	"github.com/Dynatrace/dynatrace-operator/src/registry"
 	"github.com/Dynatrace/dynatrace-operator/src/registry/mocks"
-	"github.com/containers/image/v5/docker/reference"
```

Comment:
@StefanHauth could you please remove it from go.mod too?

 https://github.com/Dynatrace/dynatrace-operator/blob/5422bcfc70d52e33872070b7c8cb8dc4ecbd541e/go.mod#L7